content/using_ec2_spot_instances_with_eks/scaling/test_hpa.md

```diff
@@ -0,0 +1,115 @@
+---
+title: "Scale the Application"
+date: 2018-08-07T08:30:11-07:00
+weight: 50
+---
+
+We are now ready to test the Horizontal Pod Autoscaler ! As a bonus we will also see the interaction between Cluster Autoscaler and Horizontal Pod Autoscaler.
+
+### Deploying the Stress CLI to Cloud 9
+
+To help us stress the application we will install a python helper app. The python helper application just calls in parallel on multiple process request to the monte-carlo-pi-service. This should generate load in our pods, which should also trigger the Horizontal Pod Autoscaler action for scaling the monte-carlo-pi-service replicaset.
```

Comment:
`should` why so shy?

Reply:
I've been trying to understand https://arxiv.org/pdf/1208.1189.pdf to explain how Spot instance fit to build Anti-fragile systems and prove it mathematecally...  My assertiveness is down to very low levels. That must be it for the "should".

I'll change it for "this will generate load", etc. Thanks !